release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1924,6 +1932,22 @@ In the following tables, features are marked with the following statuses:
 
 * There is a known issue with user-defined networks (UDN) that causes OVN-Kubernetes to delete any routing table ID equal or higher to 1000 that it does not manage. Consequently, any Virtual Routing and Forwarding (VRF) instance created outside OVN-Kubernetes is deleted. This issue impacts users who have created user-defined VRFs with a table ID greater than or equal to 1000. As a workaround, users must change their VRFs to a table ID lower than 1000 as these are reserved for {product-title}. (link:https://issues.redhat.com/browse/OCPBUGS-50855[*OCPBUGS-50855*])
 
+* There is a known issue in {product-title} version 4.18 that prevents configuring multiple subnets in the failure domain of a Nutanix cluster during installation.
+There is no workaround for this issue.
+(link:https://issues.redhat.com/browse/OCPBUGS-49885[*OCPBUGS-49885*])
+
+ * The following known issues with configuring multiple subnets for an existing Nutanix cluster exist in {product-title} version 4.18:
++
+-- 
+** Adding subnets above the existing subnet in the `subnets` stanza causes a control plane node to become stuck in the `SchedulingDisabled` state.
+As a workaround, only add subnets below the existing subnet in the `subnets` stanza.
+
+** Sometimes, after adding a subnet, the updated control plane machines appear in the Nutanix console but the {product-title} cluster is unreachable.
+There is no workaround for this issue.
+-- 
++
+(link:https://issues.redhat.com/browse/OCPBUGS-50904[*OCPBUGS-50904*]) 
```

Comment:
@huali9 and @yanhua121 here are my suggestions for known issue text

Reply:
Thank you @jeana-redhat Same with https://github.com/openshift/openshift-docs/pull/88405#discussion_r1960822834 How about changing "causes a control plane node to become stuck in the `SchedulingDisabled` state" to "causes a control plane machine to become stuck in the `Deleting` state"